include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
Why do you need to suppress this warning? I don't see it in our nightlies (e.g. http://cdash.seqan.de/viewBuildError.php?type=1&buildid=138468) and is not specific to VS2015, right?